steamworks/__init__.py

```
@@ -0,0 +1,217 @@
+        if result == 2:
+            raise SteamNotRunningException('Steam is not running')
+
+        elif result == 3:
+            raise SteamConnectionException('Not logged on or connection to Steam client could not be established')
+
+        elif result != 0:
+            raise GenericSteamException('Failed to initialize STEAMWORKS API')
+
+        return True
+
+    def relaunch(self, app_id: int) -> bool:
```

Ruff (0.14.5)

Unused method argument: `app_id`

(ARG002)